Source/SwiftLintFramework/Helpers/RegexHelpers.swift

```diff
@@ -8,7 +8,7 @@
 
 struct RegexHelpers {
     // A single variable
-    static let varName = "[a-zA-Z_][a-zA-Z0-9_]+"
+    static let varName = "[a-zA-Z_\.][a-zA-Z0-9_\.]+"
```

Comment:
I don't think you should need to add `.` in the first range, as a variable can't start with `.`